components/Input/breakSensors.py

```diff
@@ -2,21 +2,22 @@
 import logging as log
 
 class SensorKey(IntEnum):
-    kLoadingSensor = 3
-    kShootingSensor = 0
+    kLoadingSensor = 0
+    kHopperSensor = 1
+    kShootingSensor = 2
 
 class State:
     kTripped = False
     kNotTripped = True
 
 class Sensors:
 
-    compatString = ["doof", "testBoard"]
+    compatString = ["doof", "testBoard", "teapot"]
     digitalInput_breaksensors: dict
     SensorArray = []
 
     def on_enable(self):
-        for x in range(4):
+        for x in range(3):
```

Comment:
Instead of hard coded 3 do we have the ability to read from the config and do a length?